<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>administratoranlegen</class>
 <widget class="QDialog" name="administratoranlegen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="txtlbl_administratoranlegen">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Administrator anlegen</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtlbl_adminmail">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Email des Admins</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtlbl_adminpw">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Passwort des Admins</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtlbl_adminvorname">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Vorname</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtlbl_adminnachname">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nachname</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtlbl_admingeburtstag">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Geburtstag</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_email">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_passwort">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_vorname">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nachname">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_geburtstag">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_administratoranlegenundweiter">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>330</y>
     <width>331</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Administrator anlegen und weiter zur Startseite</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_programmbeenden">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>330</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Programm beenden</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
